### DISCUSSION TITLE
Welcome to win32metadata Discussions!

### DISCUSSION THREAD

**Discussion by sotteson1** [microsoft/win32metadata#258] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AArnott** [comment]
Hey gang! I'm the owner of the [C# projection](https://github.com/microsoft/CsWin32) of this metadata.

**Comment by kennykerr** [comment]
Hi folks, I'm the creator of the **Rust** projection of this metadata - the `windows` crate spans both Win32 and WinRT metadata, letting you call any Windows API past, present, and future. 

Repo: https://github.com/microsoft/windows-rs
Crate: https://crates.io/crates/windows
